x/foundation/keeper/keys.go

```diff
@@ -0,0 +1,113 @@
+package keeper
+
+import (
+	"encoding/binary"
+	"time"
+
+	sdk "github.com/line/lbm-sdk/types"
+)
+
+// Keys for foundation store
+// Items are stored with the following key: values
+var (
+	paramsKey         = []byte{0x00}
+	foundationInfoKey = []byte{0x01}
+
+	// treasuryKey = []byte{0x10}
+
+	validatorAuthKeyPrefix = []byte{0x20}
+
+	memberKeyPrefix          = []byte{0x30}
+	previousProposalIDKey    = []byte{0x31}
+	proposalKeyPrefix        = []byte{0x32}
+	proposalByVPEndKeyPrefix = []byte{0x33}
+	voteKeyPrefix            = []byte{0x34}
+)
+
+// Uint64FromBytes converts a byte array to uint64
+func Uint64FromBytes(bz []byte) uint64 {
+	return binary.BigEndian.Uint64(bz)
+}
+
+// Uint64ToBytes converts a number in uint64 to a byte array
+func Uint64ToBytes(number uint64) []byte {
+	bz := make([]byte, 8)
+	binary.BigEndian.PutUint64(bz, number)
+	return bz
+}
+
+// validatorAuthKey key for a specific validator from the store
+func validatorAuthKey(valAddr sdk.ValAddress) []byte {
+	key := make([]byte, len(validatorAuthKeyPrefix)+len(valAddr))
+	copy(key, validatorAuthKeyPrefix)
+	copy(key[len(validatorAuthKeyPrefix):], valAddr)
+	return key
+}
+
+// memberKey key for a specific member from the store
+func memberKey(address sdk.AccAddress) []byte {
+	key := make([]byte, len(memberKeyPrefix)+len(address))
```

Comment:
We don't need to store the length of the address which comes last, because we can derive it by subtracting the length of the previous bytes from the entire length.

Reply:
I wrote a review for the unity of the key storage method, but when I think about it, the gas will increase as the length of the key gets longer... Then I think your method is better :)